voxel51/users/api.py

```diff
@@ -458,6 +458,30 @@ def delete_data(self, data_id):
         res = self._requests.delete(endpoint, headers=self._header)
         _validate_response(res)
 
+    def batch_update_data_ttl(self, data_ids, days):
+        '''Updates the expiration date of all specified data.
+
+        Args:
+            data_ids (list): the data IDs
+            days (float): the number of days by which to extend the lifespan
+                of the data
+
+        Raises:
+            :class:`APIError` if the request was unsuccessful
+        '''
+        return self._batch_request("data", "ttl", data_ids, {"days": days})
+
+    def batch_delete_data(self, data_ids):
```

Comment:
can we delete `batch_` from these method names and just prepend an `s` instead?

`delete_datas` is a bit funny, but all the rest would be naturally pluralizable.

The justification for this would be that auto-complete would show you every way to delete something when you type `delete_ <tab>`.

Reply:
Counterpoint: `batch_<tab>` shows you every available batch action.
I chose this prefix for consistency (and to avoid `datas`), as well as making it clear that these are separate functions. IMO, the difference between `update_data_ttl` and `update_data_ttls` isn't as immediately obvious.

Reply:
fair enough. I'm with you